### PR TITLE
Angeline assignment2

### PR DESCRIPTION
**Pull request by angelinetu**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
pulled post data from supabase; fixed styling 

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ronniebeggs

### CODE REVIEW

File: .env
Comment (ronniebeggs) [inline .env:1]: Make sure you don't commit `.env` files! Add `.env` to you `.gitignore` file.

File: src/screens/PostScreen.tsx [hunk 1]
```diff
@@ -1,43 +1,117 @@
...
+  }, []);
+
+  const fetchPostData = async () => {
+    try {
+      const data = await getPostData();
+      setPostData(data);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx:44]: I don't know if you necessarily need a `fetchPostData` and `getPostData` function in the same file. Usually you'd store the `getPostData` function in a `/supabase/post.ts` queries file.

File: src/screens/PostScreen.tsx [hunk 2]
```diff
@@ -1,43 +1,117 @@
...
+                <View style={styles.commentHeader}>
+                  <Text style={styles.commentName}>philip_ye</Text>
+                  <Text style={styles.commentDate}>September 20</Text>
+                </View>
+                <Text style={styles.commentText}>
+                  This organization is doing amazing work tackling the complex
+                  root causes of the issue.
+                </Text>
+              </View>
+            </View>
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx:99]: Given that there are so many divs for each div, might be helpful to create a separate `Comment` component.

There's also a comments table that you can pull this data from!

File: src/screens/styles.ts [hunk 3]
```diff
@@ -4,7 +4,103 @@ export const styles = StyleSheet.create({
...
+  },
+  likesText: {
+    marginLeft: 5,
+  },
+  commentSection: {
+    borderTopWidth: 1,
+    borderTopColor: '#ccc',
+    paddingTop: 10,
+    marginTop: 20,
+    paddingRight: 10,
```
Comment (ronniebeggs) [inline src/screens/styles.ts:64]: Avoid asymmetric spacing attributes. See if you can achieve this styling using flexbox properties.

File: src/screens/styles.ts [hunk 4]
```diff
@@ -4,7 +4,103 @@ export const styles = StyleSheet.create({
...
+    justifyContent: 'space-between',
+  },
+  profileName: {
+    fontWeight: 'bold',
+    fontSize: 16,
+    marginLeft: 8,
+  },
+  postDate: {
+    color: 'gray',
+    paddingLeft: 120,
```
Comment (ronniebeggs) [inline src/screens/styles.ts:23]: You should be able to create an offset like this using `justifyContent` and `alignItems` attributes.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Looking good so far! I left a few comments/changes. For furture reference, make sure to write more in-depth PR descriptions.
